src/kaltura-player.js

```diff
@@ -794,6 +797,15 @@ class KalturaPlayer extends FakeEventTarget {
   get crossOrigin(): ?string {
     return this._localPlayer.crossOrigin;
   }
+
+  /**
+   * Gets the player tab visibility state
+   * @returns {boolean} - whether the browser player tab is visible
+   * @public
+   */
+  get isTabVisible(): boolean {
+    return this._localPlayer.isTabVisible;
```

Comment:
See playkit-js review, don't think this level of detail is required. Check with Kava if tab is also relevant for scroll.
Also need to expose some API for visibility beside tab, probably isVisibile is enough for now.

Reply:
In KAVA we left the old code of checking tab (until prop will be removed) so this API was removed